src/Rdap.php

```diff
@@ -66,9 +121,30 @@ public function dns(): RdapDns
     {
         return $this->rdapDns;
     }
+    public function ipv4(): RdapIpV4
+    {
+        return $this->rdapIpV4;
+    }
+    public function ipv6(): RdapIpV6
+    {
+        return $this->rdapIpV6;
+    }
+    
 
     public function supportedTlds(): array
     {
         return $this->dns()->supportedTlds();
     }
+    private function getIpAndVersion(string $ip): ?array
```

Comment:
Make this `protected`

This function returns both the version and ip. The returned ip is the same as the one given. Let's make it simpler by just returning the version.

Could you also make an enum `IpVersion` for the version and let the function return the enum?